pastas/model.py

```diff
@@ -1239,6 +1239,54 @@ def get_transform_contribution(self, tmin=None, tmax=None):
         sim_org = ml.simulate(tmin=tmin, tmax=tmax)
         return sim - sim_org
 
+    def get_output_series(self, tmin=None, tmax=None, split=True):
+        """Method to get all the modeled time series from the pastas Model.
+
+        Parameters
+        ----------
+        tmin: str, optional
+            String with a start date for the simulation period (E.g. '1980').
+            If none is provided, the tmin from the oseries is used.
+        tmax: str, optional
+            String with an end date for the simulation period (E.g. '2010').
+            If none is provided, the tmax from the oseries is used.
+        split: bool, optional
+            Passed on to ml.get_contributions. Split the contribution from
+            recharge into evaporation and precipitation. See also
+            ml.get_contributions.
+
+        Returns
+        -------
+        series: pandas.DataFrame
```

Comment:
Variable name `series` for a type `pandas.DataFrame` confused me when reviewing.. But I guess this is not the end of the world.

Reply:
changed to df